docassemble/AssemblyLine/data/static/al_audio.js

```diff
@@ -1,5 +1,14 @@
 $(document).on('daPageLoad', function() {
-  al_js.audio_minimal_controls( $('.daaudio-control')[0], 'page_reader_substitute' );
+  var $audio_nodes = $('.daaudio-control');
+  var id_count = 1;
+  for ( var audio_node of $audio_nodes ) {
+    (function () {
+      console.log( audio_node.constructor.name );
+      al_js.replace_with_audio_minimal_controls( audio_node, 'page_reader_substitute_' + id_count );
+    })();
```

Comment:
minor: I've sometimes seen this "wrap everything in a function then execute immediately" pattern, but I don't really like it over just executing the statements. I can't tell, but is there some reason it needs to be like this? It seems to work fine without it, but there could be corner cases I don't know about.

Reply:
Ah, it's possible I was just having a brain freeze. There are [some situations in which it is necessary](https://stackoverflow.com/a/19586183/14144258). For example, when you're putting an event listener on a list of elements and you can only use `var`:

```js
for ( var item of list ) { item.addEventListener('click', function(){ console.log( 'log' ) }); }
```

The above will put the event listener only on the last element. Usually these days people use `let` instead, but our aim to be compatible with older browsers doesn't allow that. Regardless, this loop is already just calling a function, so that shouldn't be a problem here.

I also left the console log in there accidentally. I can clean it all up.

Reply:
Didn't realize we were avoid `let` because of older browsers, good to know!